cluster/gce/gci/apiserver-helper.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors.
+# Copyright 2020 Authors of Arktos - file modified.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A library of helper functions and constant for GCI distro
+source "${KUBE_ROOT}/cluster/gce/gci/helper.sh"
+
+# create-apiserver-instance creates the apiserver instance. If called with
+# an argument, the argument is used as the name to a reserved IP
+# address for the apiserver. (In the case of upgrade/repair, we re-use
+# the same IP.)
+#
+# It requires a whole slew of assumed variables, partially due to to
+# the call to write-apiservr-env. Listing them would be rather
+# futile. Instead, we list the required calls to ensure any additional
+#
+# variables are set:
+#   ensure-temp-dir
+#   detect-project
+#   get-bearer-token
+function create-apiserver-instance {
+  local name=""
+  local address=""
+  local private_network_ip=""
+  [[ -n ${2:-} ]] && address="${2}"
```

Comment:
The 3 lines is to handle the 3 parameters. Why the order is 2,1,3 instead of 1,2,3?